ShaderConverter/ShaderConv/context.cpp

```diff
@@ -3693,6 +3693,7 @@ CContext::EmitSrcOperand( const CInstr& instr,
     if (dwRegType == D3DSPR_TEXTURE)
     {
         const DWORD dwRegNum = D3DSI_GETREGNUM_RESOLVING_CONSTANTS(dwToken) + dwOffset;
+        UNREFERENCED_PARAMETER(dwRegNum);

```

Comment:
Same as `D3D12TranslationLayer` comments, for these assert-only constants, prefer `[[maybe_unused]]` before `UNREFERENCED_PARAMETER` (which is also wrong, there's a different macro for suppressing this warning on local variables, though obviously they both work since they're just `(void)var;` at the end of the day).

Reply:
Fixed.